manage_breast_screening/mammograms/jinja2/mammograms/show/appointment_details.jinja

```diff
@@ -36,6 +37,8 @@
 
   {{ appointment_header(presented_appointment) }}
 
+  {{ special_appointment_banner(presented_appointment.special_appointment, show_change_link=False) }}
```

Comment:
Do we want to hide the change link here? The prototype has a change link on this page.

Reply:
This is the completed state - it doesn't make sense to change the special appointment for an appointment that is complete